### ISSUE TITLE
Webpack Compiling Error afte update 5.5.0

### ISSUE THREAD

**Issue by andrei22b** [cypress-io/cypress#8982]
### Current behavior
When trying to open Cypress 5.5.0 after updating i get the error: `Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/***/***/tests/e2e/webpack.config.js`

### Full Error
```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/-/-/tests/e2e/webpack.config.js
require() of ES modules is not supported.
require() of /Users/-/-/tests/e2e/webpack.config.js from /Users/-/-/tests/e2e/utils/index.ts is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename webpack.config.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /Users/-/-/tests/e2e/package.json.

    at Module._extensions..js (internal/modules/cjs/loader.js:1162:13)
    at Object.require.extensions.<computed> [as .js] (/Users/-/Library/Caches/Cypress/5.5.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/ts-node/src/index.ts:539:12)
    at Module.load (internal/modules/cjs/loader.js:981:32)
    at Module._load (internal/modules/cjs/loader.js:881:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:1023:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/Users/-/-/tests/e2e/utils/index.ts:5:1)
    at Module._compile (internal/modules/cjs/loader.js:1145:30)
    at Module.m._compile (/Users/-/Library/Caches/Cypress/5.5.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/ts-node/src/index.ts:536:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1166:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/-/Library/Caches/Cypress/5.5.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/ts-node/src/index.ts:539:12)
    at Module.load (internal/modules/cjs/loader.js:981:32)
    at Module._load (internal/modules/cjs/loader.js:881:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:1023:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at runPlugins (/Users/-/Library/Caches/Cypress/5.5.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/run_plugins.js:213:15)
    at Object.<anonymous> (/Users/-/Library/Caches/Cypress/5.5.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/index.js:8:25)
    at Module._compile (internal/modules/cjs/loader.js:1145:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1166:10)
    at Module.load (internal/modules/cjs/loader.js:981:32)
```
### Versions

5.5.0

### Webpack config

```js
const path = require('path');

module.exports = {
    stats: 'detailed',
    module: {
        rules: [
            {
                test: /\.tsx?$/,
                use: 'ts-loader',
                exclude: /node_modules/
            }
        ]
    },
    resolve: {
        extensions: ['.tsx', '.ts', '.js']
    },
    output: {
        filename: 'bundle.js',
        path: path.resolve(__dirname, 'dist')
    },
    node: {
        fs: 'empty',
        net: 'empty',
        tls: 'empty'
    }
};
```

**Comment by jennifer-shehane** [comment]
Could you clarify what Cypress version you were updating from? What was the last working version?

Looks like this could be #8090

**Comment by andrei22b** [comment]
Last version was 5.3.0

**Comment by davidtorge12** [comment]
Same problem here,
the error appeared starting with version 5.4. I thought that it will be fixed in 5.5 but it seems that its persist

**Comment by andrei22b** [comment]
Fixed with renaming **webpack.config.js** to **webpack.config.cjs**
